build.rs

```diff
@@ -34,4 +39,29 @@ fn main() {
     content.truncate(36);
     let aaguid = Uuid::parse_str(&content).unwrap();
     aaguid_bin_file.write_all(aaguid.as_bytes()).unwrap();
+
+    // COSE encoding the public key, then write it out.
+    let file = File::open("crypto_data/opensk_upgrade_pub.pem").unwrap();
+    let (pem, _bytes_read) = Pem::read(std::io::BufReader::new(file)).unwrap();
```

Comment:
I would rather use OpenSSL to manipulate and extract these more reliably.
If you look in the history (e.g. https://github.com/google/OpenSK/blob/6c363981c4b8b7a09716095e2b93043ef2a35c8a/build.rs), I used to do it to validate all keys/certificate.

Reply:
I now parse the file using OpenSSL. Makes sense given that OpenSSL is also used earlier in the stack. It's different from your code, given that I parse an uncompressed public key instead of a private key. PTAL